src/dictionary/dictionary.c

```diff
@@ -255,7 +254,7 @@ dictionary_open(
 		ion_dictionary_t			fallback_dict;
 		ion_err_t					err;
 
-		ffdict_init(&fallback_handler);
+		/*ffdict_init(&fallback_handler);*/
```

Comment:
Uncomment these lines - this should've worked. The reason why it broke, is because we're not linking the flat file library anymore. The proper solution is to remove the inclusion references (seen further down) that reference this file - since we don't use it, we don't need to include it.